docs/layouts/shortcodes/generated/core_configuration.html

```diff
@@ -1050,6 +1050,12 @@
             <td>Integer</td>
             <td>Write batch size for any file format if it supports.</td>
         </tr>
+        <tr>
+            <td><h5>write.skip-actions</h5></td>
+            <td style="word-wrap: break-word;">(none)</td>
+            <td>String</td>
+            <td>This parameter only works when write-only is false., You can specify which actions to skip during the write process.<br />1. 'partition-expire': Skip the action of partition expiration.<br />2. 'snapshot-expire': Skip the action of snapshot expiration.<br />3. 'create-tag': Skip automatic tag creation.<br />Both can be configured at the same time: 'partition-expire,snapshot-expire,create-tag'.</td>
```

Comment:
I prefer to add a separate enable conf for each such post-commit behavior, with the default set to true.

Reply:
Thanks for review, If the parameters are separated, will it be more cumbersome to use ? For my needs, I need to skip `partition-expire` & `snapshot-expire` & `create tag` but retain `compact`, so I may need to configure three parameters to meet my needs.

Reply:
Maybe you can add a `write-actions`? Default is 'all'. The values can be:
`snapshot-expire`, `partition-expire`, `compaction`, `tag-automatic-creation`.

Reply:
Does this `compact` mean `full` or `minor` or both, or should it be separated into two actions `full_compact` and `minor_compact`?

Reply:
maybe full or minor should be separate option.

Reply:
Ok, I will fix it.